packages/testcontainers/src/containers/RegTestContainer/Masternode.ts

```diff
@@ -1,23 +1,22 @@
 import { MasterNodeKey, RegTestFoundationKeys } from '@defichain/jellyfish-network'
-import { waitForCondition } from '../../utils'
 import { DockerOptions } from 'dockerode'
 import { DeFiDContainer, StartOptions } from '../DeFiDContainer'
 import { RegTestContainer } from './index'
+import { NativeChainWaitFor } from '../../index'
```

Comment:
Considering you're also exporting `Masternode` in `../../index.ts`, I feel like this should've caused a circular dependency issue.

A quick check using `madge` reveals it as well:

```
❯ madge . --circular
Processed 17 files (294ms) 

✖ Found 11 circular dependencies!

1) dist/containers/AppContainer/WhaleSanityContainer.js > dist/containers/AppContainer/index.js > dist/containers/RegTestContainer/Masternode.js > dist/containers/DeFiDContainer.js > dist/index.js
2) dist/containers/DeFiDContainer.js > dist/index.js
3) dist/containers/DeFiDContainer.js > dist/index.js > dist/containers/MainNetContainer.js
4) dist/containers/NativeChainContainer.js > dist/containers/NativeChainRpc.js
5) dist/index.js > dist/containers/NativeChainContainer.js > dist/containers/NativeChainRpc.js
6) dist/containers/RegTestContainer/Masternode.js > dist/containers/DeFiDContainer.js > dist/index.js > dist/containers/RegTestContainer/LoanContainer.js
7) dist/containers/RegTestContainer/Masternode.js > dist/containers/DeFiDContainer.js > dist/index.js
8) dist/containers/DeFiDContainer.js > dist/index.js > dist/containers/RegTestContainer/Persistent.js
9) dist/containers/RegTestContainer/Masternode.js > dist/containers/DeFiDContainer.js > dist/index.js > dist/containers/RegTestContainer/Persistent.js
10) dist/containers/DeFiDContainer.js > dist/index.js > dist/containers/RegTestContainer/index.js
11) dist/containers/DeFiDContainer.js > dist/index.js > dist/containers/TestNetContainer.js
```

A pretty simple fix are to remove this imports from `index.ts` where you export your file, and the circular dep problem will be gone.